assets/js/custom.js

```diff
@@ -2,50 +2,65 @@ var custom = {
     loadTasks: function(numSubtasks) {
         /*
          * This function is called on page load and should implement the promise interface
-         * The final data returned should be an array of objects with length 
-         * config.meta.numTasks, one object for each task
+         * If aggregate is set to false, the final data returned should be an array of objects 
+         * with length config.meta.numTasks, one object for each task. 
```

Comment:
what if `aggregate` is set to true? also maybe write "if `config.meta.aggregate`?